### PR TITLE
Store the parsed errors from response in an attribute for checking when handling exceptions

### PR DESCRIPTION
**Pull request by anero**
…en handling exceptions

### CODE REVIEW

File: lib/ngp_van/error.rb [hunk 1]
```diff
@@ -45,23 +45,20 @@ def initialize(response = nil)
...
-    end
-
     def build_error
       return nil if response.nil?
 
       {
         status: status,
         errors: errors
-      }
+      }.to_s
```
Comment (woodhull) [inline lib/ngp_van/error.rb]: Why this change?

Comment (anero) [reply lib/ngp_van/error.rb]: `Exception` (`StandardError` base class) constructor expects a string representing the message: https://ruby-doc.org/core-2.5.0/Exception.html#method-c-new

Comment (woodhull) [reply lib/ngp_van/error.rb]: This is changing the behavior of the gem in ways that other consumers might not appreciate though. It's actually not clear to me that message should always be a string? it's not as if it's type checking. 

I think that if I were designing the gem from scratch raising with an informative message string would be my choice, but I do not see why a hash would actually break anything (and it's the current behavior right now)

**Comment by woodhull** [comment]
Other than the `to_s` bit, this seems good.

File: lib/ngp_van/error.rb (hunk 1 above)
Comment (anero) [reply lib/ngp_van/error.rb]: OK, I will switch back to return a `Hash`. 
I'm not familiar enough with ruby's C implementation to say for sure, but when running the code, `Exception` seems to be effectively calling `to_s` on the argument and given the standard library documentation says the argument is a _message_, I think it makes more sense to pass a string.
